packages/mobile/src/paymentRequest/OutgoingPaymentRequestListScreen.tsx

```diff
@@ -0,0 +1,115 @@
+import OutgoingPaymentRequestListItem from 'src/paymentRequest/OutgoingPaymentRequestListItem'
+import PaymentRequestBalance from 'src/paymentRequest/PaymentRequestBalance'
+import PaymentRequestListEmpty from 'src/paymentRequest/PaymentRequestListEmpty'
+import { getRecipientFromPaymentRequest } from 'src/paymentRequest/utils'
+import { NumberToRecipient } from 'src/recipients/recipient'
+import { recipientCacheSelector } from 'src/recipients/reducer'
+import { RootState } from 'src/redux/reducers'
+import DisconnectBanner from 'src/shared/DisconnectBanner'
+
+const { contentPadding } = variables
+
+interface StateProps {
+  dollarBalance: string | null
+  paymentRequests: PaymentRequest[]
+  e164PhoneNumberAddressMapping: E164NumberToAddressType
+  recipientCache: NumberToRecipient
+}
+
+interface DispatchProps {
+  updatePaymentRequestStatus: typeof updatePaymentRequestStatus
+  updatePaymentRequestNotified: typeof updatePaymentRequestNotified
+  fetchPhoneAddresses: typeof fetchPhoneAddresses
+}
+
+const mapStateToProps = (state: RootState): StateProps => ({
+  dollarBalance: state.stableToken.balance,
+  paymentRequests: getOutgoingPaymentRequests(state),
+  e164PhoneNumberAddressMapping: e164NumberToAddressSelector(state),
+  recipientCache: recipientCacheSelector(state),
+})
+
+type Props = WithNamespaces & StateProps & DispatchProps
+
+export class OutgoingPaymentRequestListScreen extends React.Component<Props> {
+  static navigationOptions = () => ({
+    ...headerWithBackButton,
+    headerTitle: i18n.t('paymentRequestFlow:outgoingPaymentRequests'),
+  })
+
+  renderRequest = (request: PaymentRequest, key: number, allRequests: PaymentRequest[]) => {
+    const { recipientCache } = this.props
+    const requester = getRecipientFromPaymentRequest(request, recipientCache)
+
+    return (
+      <View key={key}>
+        <OutgoingPaymentRequestListItem
+          id={request.uid || ''}
+          amount={request.amount}
+          updatePaymentRequestStatus={this.props.updatePaymentRequestStatus}
+          updatePaymentRequestNotified={this.props.updatePaymentRequestNotified}
+          requester={requester}
+          comment={request.comment}
+        />
+        {key < allRequests.length - 1 && <View style={styles.separator} />}
+      </View>
+    )
+  }
+
+  render() {
+    return (
+      <SafeAreaView style={styles.container}>
+        <DisconnectBanner />
+        <PaymentRequestBalance dollarBalance={this.props.dollarBalance} />
+        {this.props.paymentRequests.length > 0 ? (
+          <ScrollView>
+            <View style={[componentStyles.roundedBorder, styles.scrollArea]}>
+              {this.props.paymentRequests.map(this.renderRequest)}
+            </View>
+          </ScrollView>
+        ) : (
+          <PaymentRequestListEmpty />
+        )}
+      </SafeAreaView>
+    )
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: colors.background,
+    flex: 1,
+  },
+  separator: {
```

Comment:
There's a HorizontalLine component in react-components/components for this

Reply:
It has unchangeable `width: '100%'`. I could've make it changeable, but on the new design this line will disappear anyway.